src/Uno.UI.Wasm/ts/Windows/System/Display/DisplayRequest.ts

```diff
@@ -0,0 +1,36 @@
+﻿interface Navigator {
+	wakeLock : WakeLock;
+}
+
+enum WakeLockType {
+	screen = "screen"
+};
+
+interface WakeLock {
+	request(type: WakeLockType): Promise<WakeLockSentinel>;
+};
+
+interface WakeLockSentinel {    
+	release(): Promise<void>;	
+};
+
+namespace Windows.System.Display {
+
+	export class DisplayRequest {
+		private static activeScreenLockPromise: Promise<WakeLockSentinel>;
+
+		public static activateScreenLock() {
+			if (navigator.wakeLock) {
+				DisplayRequest.activeScreenLockPromise = navigator.wakeLock.request(WakeLockType.screen);
+				DisplayRequest.activeScreenLockPromise.catch(reason => console.log("Could not acquire screen lock (" + reason + ")"));
```

Comment:
Codacy found an issue: [Exceeds maximum line length of 120](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5767548)